src/mame/drivers/volfied.cpp

```diff
@@ -211,28 +210,54 @@ GFXDECODE_END
 
 void volfied_state::machine_start()
 {
-	cchip_init();
 }
 
 void volfied_state::machine_reset()
 {
-	cchip_reset();
 }
 
+TIMER_DEVICE_CALLBACK_MEMBER(volfied_state::scanline)
+{
+	if (param == 224)
```

Comment:
```
MCFG_SCREEN_SIZE(320, 256)
MCFG_SCREEN_VISIBLE_AREA(0, 319, 8, 247)

```
vblank definitely can't be triggered at 224, should rather be at 248 according to the screen def. And c-chip is probably in between 248 and 8 and assuming there isn't a clear irq signal sent from one of the command latches.

Reply:
c-chip must at least happen after the vbl irq has processed the inputs, otherwise it ends up reading it 'mid-state' and causing things to go badly wrong, but yeah, the values used were just for testing.